core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackServiceTaskManager.java

```diff
@@ -176,6 +177,19 @@ public synchronized void disableSleepTimer() {
         }
     }
 
+    /**
+     * Enables the sleep timer.
+     */
+    public synchronized void enableSleepTimer() {
+        if (!isSleepTimerActive()) {
```

Comment:
Can't you just call `setSleepTimer(SleepTimerPreferences.timerMillis())` instead of adding a new function?